solutions/usaco-714.mdx

```diff
@@ -238,3 +241,57 @@ int main() {
 	ps(ans);
 }
 ```
+</CPPSection>
+
+<JavaSection>
+```java
+import java.util.*;
+import java.io.*;
+
+public class WhyDidTheCowCrossTheRoad {
+	public static void main (String[] args) throws IOException {
+		Kattio io = new Kattio("helpcross");
+
+		int numChickens = io.nextInt();
+		int numCows = io.nextInt();
+
+		int[] chicken = new int[numChickens]; // where the chicken are located
+		int[] a = new int[numCows];
+		int[] b = new int[numCows];
+		int[] marked = new int[numCows];
```

Comment:
if i'm reading the code correctly can't this just be a bool array instead of an int array